test/cucumber/cucumber-v4/src/test/java/com/automatedtest/sample/test/TestSteps.java

```diff
@@ -19,6 +19,7 @@
 import com.epam.drill.plugins.test2code.api.*;
 import com.epam.drill.test.agent.instrumentation.testing.cucumber.*;
 import com.epam.drill.test.common.*;
+import com.epam.drill.test.common.TestData;
```

Comment:
why do you add?

Reply:
In the new version of the t2 api, a class with exactly the same name appeared, so we had to explicitly import this class